src/ng/components/body/body.core.js

```diff
@@ -58,41 +72,62 @@ class BodyCore extends Directive(BODY_CORE_NAME, {view: `^^${VIEW_CORE_NAME}`})
 		}
 	}
 
-	toggle(cell) {
+	onMouseDown(e) {
+		if (!this.isRange) {
+			return;
+		}
+
+		this.rangeStart = e;
+	}
+
+	onMouseMove(e) {
+		if (!this.isRange) {
+			return;
+		}
+
+		if (this.rangeStart) {
+			this.view.overlay.show();
```

Comment:
if it already shown we don't need to call it, in your case user of method know the logic of show (css visibility)